runtime/gc_glue_java/ObjectModel.hpp

```diff
@@ -562,6 +562,20 @@ class GC_ObjectModel : public GC_ObjectModelBase
 	{
 		GC_ObjectModelBase::fixupForwardedObject(forwardedHeader, destinationObjectPtr, objectAge);
 
+#if defined(J9VM_ENV_DATA64)
+		if (_javaVM->isIndexableDataAddrPresent) {
```

Comment:
it's better to check if offheap enabled (to not waste time for offheap disabled)